hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MetricsRegionServer.java

```diff
@@ -53,6 +54,8 @@
   private Timer bulkLoadTimer;
   private Meter serverReadQueryMeter;
   private Meter serverWriteQueryMeter;
+  protected long slowMetricTime = 1000L;
```

Comment:
Since we will init it later, so not need give it a special value here?

Reply:
updated, please review again! thanks a lot